templates/Cloudwatch/cloudwatch-log.yaml

```diff
@@ -0,0 +1,62 @@
+Description: Setup cloudwatch logs
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  RetentionInDays:
+    Type: Number
+    Description: >-
+      The number of days to retain the log events in the specified log group. Possible values are:
+      1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, and 3653.
+    Default: 90
+    MinValue: 1
+    MaxValue: 3653
```

Comment:
Let's use this value

Reply:
do you mean use 3653 as the default?  this is a parameter which means you can override it when you deploy this template.  i think default of 90 is better than max.  you can set to max when you deploy.

Reply:
Agreed